composer.json

```
@@ -24,7 +24,38 @@
 
         "sylius/registry": "^1.0",
         "sylius/resource-bundle": "^1.0",
-        "symfony/framework-bundle": "~3.0|~4.0"
+
+        "symfony/asset": "^3.4|^4.1",
+        "symfony/config": "^3.4|^4.1",
+        "symfony/console": "^3.4|^4.1",
+        "symfony/dependency-injection": "^3.4|^4.1",
+        "symfony/doctrine-bridge": "^3.4|^4.1",
+        "symfony/event-dispatcher": "^3.4|^4.1",
+        "symfony/expression-language": "^3.4|^4.1",
+        "symfony/filesystem": "^3.4|^4.1",
+        "symfony/finder": "^3.4|^4.1",
+        "symfony/form": "^3.4|^4.1",
+        "symfony/framework-bundle": "^3.4|^4.1",
```

Why adding all this components? It won't be enough with only adding this?

```suggestion
        "symfony/framework-bundle": "^3.4|^4.1",
```

---

Ah sorry, I actually copied too much. will adjust it and let you know

---

List minified.